### ISSUE TITLE
team page: add alt tags to team images

### ISSUE THREAD

**Issue by discoform** [wurmlab/wurmlab.github.io#118]
Alt tags are needed for Accessibility and SEO performance on[ the team page](https://wurmlab.com/team/
)

# Expected Behavior

The page should score high (ideally the score should be 100) when running any url through [Lighthouse](https://web.dev/measure/)

# Current Behavior

The team page is being flagged for accessibility and SEO errors due to missing alt tags

# Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Copy the link `https://wurmlab.com/team/`
2. In a browser, go to [https://web.dev/measure/](https://web.dev/measure/)
3. Paste the url `https://wurmlab.com/team/` and Run Audit
4. View Report. It should look [something like this](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwurmlab.com%2Fteam%2F#seo)

**Comment by yannickwurm** [comment]
Hi @yuriigr  - would you be able to update the code so it automatically adds alt tags - just using the person's name "Photo of $name"?

**Comment by yuriigr** [comment]
Done.
